examples/complete/main.tf

```diff
@@ -41,6 +41,7 @@ module "backup" {
   source = "../.."
 
   backup_resources   = [module.efs.arn]
+  not_resources      = var.not_resources
```

Comment:
@stratusjerry All looks good on this, but you'll need to define the `not_resources` variable within `examples/complete/variables.tf` + define a default or fixture value otherwise our tests will fail. Can you update?

Reply:
thanks, does the update look good or should I move variable default to a **fixtures.us-east-2.tfvars** assignment?